_posts/2023-08-11-release.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Release 2023.32
+author: Priya
+date: 2023-08-11
+category: Jekyll
+layout: post
+---
+
+### Enhancements:
+
+#### SFTP Access Enhancement:
+You can now access SFTP through an SSH key for third-party imports. In addition to password-based authentication, this update introduces authentication using a private SSH key.
+
+#### Streamlined Column Copying (Alpha Feature):
+Copying multiple columns is now more efficient. Instead of creating a separate rule per column, users can use a single copy rule for multiple columns. This functionality is accessible through the transform menu and enhances workflow efficiency.
+
+
+
+
+### Defect Fixes:
+* Resolved an inconsistency in Branch out rule when deleting the resultant dataset.
+* Fixed an issue where pipeline changes were not being reflected in the data flow status modal.
+* Resolved the "Publish to DB" error that occurred.
+* Fixed the continuous loader problem during the application of copy column rule through pasting.
```

Comment:
Resolved an issue with copy paste of copy column rule.

Reply:
fixed